### ISSUE TITLE
[BUG] Fallback to documentation homepage when clicking the breadcrumb in documentation.

### ISSUE THREAD

**Issue by abhirajshourya** [opensearch-project/oui#509]
The bug is found on the documentation page of breadcrumbs, all the sub-topics display the same behavior. It takes the user back to the homepage when clicking on any breadcrumb there. The issue is happening on all the breadcrumbs of "Limit the number of breadcrumbs", "Truncate each breadcrumb" and "Responsive".

**To Reproduce**
1. Go to [limit the number of breadcrumbs](http://localhost:8030/#/navigation/breadcrumbs#limit-the-number-of-breadcrumbs).
2. Click on any breadcrumb.
3. It will take you back to the homepage.

**Host/Environment:**
 - OS: Windows 11, Ubuntu 22.04
 - Chrome, Firefox Browser - 110

**Comment by KrooshalUX** [comment]
@abhirajshourya - this behavior actually reflects what is intended with the breadcrumb (that the breadcrumbs would point to different URLs, including the homepage of the documentation). As we are currently making many changes to this forked doc site, we have simple generic URLs right now to demonstrate that the breadcrumbs are clickable. While we work through continued edits to the documentation, the content and the URLs may become more specific. 

TLDR; Closing as this is expected behavior.
